YellowNotes.Core/Services/EmailService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Threading.Tasks;
+using System.Threading;
+using YellowNotes.Core.Email;
+using MimeKit;
+using MailKit.Net.Smtp;
+
+namespace YellowNotes.Core.Services
+{
+    public class EmailService : IEmailService
+    {
+        private readonly EmailConfiguration _emailConfiguration;
+
+        public EmailService(EmailConfiguration emailConfiguration)
```

Comment:
Does the injection of `EmailConfiguration` works? I'm not sure, please check it!